apps/backend/package.json

```diff
@@ -54,8 +56,10 @@
     "helmet": "^8.1.0",
     "keyv": "^5.5.3",
     "lodash": "^4.17.21",
-    "mongodb": "^6.20.0",
-    "mongoose": "^8.19.1",
+    "mongodb": "^6.18.0",
+    "mongoose": "^8.17.0",
```

Comment:
Is there a reason these are being downgraded?

Reply:
This was probably an accident from merging in gql, we should just undo these changes